internal/pkg/modifiers/comma_space_map_modifier.go

```diff
@@ -0,0 +1,81 @@
+package pkg
+
+import (
+	"context"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+// CommaSpaceMapModifier ensures consistent formatting of comma-separated strings in map values.
+type CommaSpaceMapModifier struct{}
+
+func (m CommaSpaceMapModifier) Description(ctx context.Context) string {
+	return "Ensures consistent formatting of comma-separated strings in map values with spaces after commas"
+}
+
+func (m CommaSpaceMapModifier) MarkdownDescription(ctx context.Context) string {
+	return "Ensures consistent formatting of comma-separated strings in map values with spaces after commas"
+}
+
+func (m CommaSpaceMapModifier) PlanModifyMap(ctx context.Context, req planmodifier.MapRequest, resp *planmodifier.MapResponse) {
+	if req.PlanValue.IsUnknown() || req.PlanValue.IsNull() {
+		return
+	}
+
+	planElements := req.PlanValue.Elements()
+	newElements := make(map[string]types.String)
+
+	// Check config format if available
+	var configFormat bool // true = spaces, false = no spaces
+	if !req.ConfigValue.IsNull() {
+		configElements := req.ConfigValue.Elements()
+		// Look at first value to determine format
+		for _, v := range configElements {
+			if str, ok := v.(types.String); ok && !str.IsNull() {
+				configFormat = strings.Contains(str.ValueString(), ", ")
+				break
+			}
+		}
+	}
```

Comment:
why do we need this btw? I think an alternative would be to just always modify the map values to be ", " separated